src/libYARP_OS/src/Network.cpp

```diff
@@ -1589,7 +1584,7 @@ class StubCarrier : public ForwardingCarrier {
             ncar = nullptr;
             return nullptr;
         }
-        return ncar;
```

Comment:
The only possible breaking change could be this one..
The motivation behind this change is that `ncar` is a sort of wrapper of the carriers that are loaded as plugin, returning `->getContent()` we have the actual carrier that has been loaded and not his wrapper.